x-pack/plugins/upgrade_assistant/__jest__/client_integration/es_deprecations/cluster_settings_deprecation_flyout.test.ts

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { act } from 'react-dom/test-utils';
+
+import { setupEnvironment } from '../helpers';
+import { ElasticsearchTestBed, setupElasticsearchPage } from './es_deprecations.helpers';
+import { esDeprecationsMockResponse, MOCK_SNAPSHOT_ID, MOCK_JOB_ID } from './mocked_responses';
+
+describe('Cluster settings deprecation flyout', () => {
+  let testBed: ElasticsearchTestBed;
+  const { server, httpRequestsMockHelpers } = setupEnvironment();
+  const clusterSettingDeprecation = esDeprecationsMockResponse.deprecations[4];
+
+  afterAll(() => {
+    server.restore();
+  });
+
+  beforeEach(async () => {
+    httpRequestsMockHelpers.setLoadEsDeprecationsResponse(esDeprecationsMockResponse);
+    httpRequestsMockHelpers.setUpgradeMlSnapshotStatusResponse({
+      nodeId: 'my_node',
+      snapshotId: MOCK_SNAPSHOT_ID,
+      jobId: MOCK_JOB_ID,
+      status: 'idle',
+    });
+    httpRequestsMockHelpers.setReindexStatusResponse({
+      reindexOp: null,
+      warnings: [],
+      hasRequiredPrivileges: true,
+      meta: {
+        indexName: 'foo',
+        reindexName: 'reindexed-foo',
+        aliases: [],
+      },
+    });
```

Comment:
nit: are those 2 mock responses for ml and reindex needed here? If not, it could be good to remove them.